Pocket/data-flows
# Configure S3 Result for braze/update_flow

# Goal
Allow us to resume flows from failure, instead of having to re-run the entire flow, by storing results on S3.

# Implementation decisions
- I deleted an unused unused S3 bucket called 'storage', which we previously used to store code, to avoid confusion.
- To lower cost, we might choose to set a bucket policy to automatically delete results. I haven't yet done this, and I'm not sure if it's material.

# Deployment
- [x] Delete `pocket-dataflows-storage-dev` S3 bucket in development
- [x] Delete `pocket-dataflows-storage-prod` S3 bucket in production

# QA
I pushed this to the dev branch, and randomly threw an exception with 10% probability:
```python
    import random
    if random.random() < 0.1:
        logger.info('Throwing a fit')
        raise Exception('Random exception')
```
![Screenshot from 2022-04-04 11-31-53](https://user-images.githubusercontent.com/1547251/161609236-8b990844-af2e-4f3e-877f-8d5df163fa92.png)

It functioned as expected:
1. In the first run 3/60 tasks failed.
2. When clicking 'Restart from failure', all tasks succeeded.
